README.md

```diff
@@ -34,6 +34,8 @@ but likely needed for production and test deployments.
 | POSTGRES_PASSWORD         | PosgreSQL database password                                                                                         | None                            |
 | POSTGRES_ADMIN_USER       | PosgreSQL database admin user                                                                                       | None                            |
 | POSTGRES_ADMIN_PASSWORD   | PosgreSQL database admin password                                                                                   | None                            |
+| REDIS_HOST       | Redis database host user                                                                                       | None                            |
```

Comment:
Add some comments regarding the usage of this Redis (what does it does) and the fact that is optional.

Reply:
fixed